packages/elements-dev-portal/src/interfaces/node.ts

```diff
@@ -0,0 +1,28 @@
+export type Node = NodeSummary & {
```

Comment:
I agree it's kind of hard to colocate this with anything, that's for sure.

But it's certainly not `interfaces`, there isn't a single `interface` in this file. What about doing what we always do, and creating a top level `types.ts`, then forward-exporting the `Node` type from `index.ts`? That is probably the only thing from here that we really need.

Reply:
Done in https://github.com/stoplightio/elements/pull/1191/commits/7b06bf468d589f7244993df9366dfe1c15a2f9a8